gem5.patch

```diff
@@ -21,7 +21,7 @@ index 0e266a6f4..1f41351ca 100644
 +                "/opt/rocm/hipblas/lib",
 +                "/opt/rocm/rocblas/lib"
             ]),
-+           "HOME=/",
++           'HOME=%s' % os.environ.get('HOME'),
```

Comment:
You should double-check that this works for you in your setup, but as far as I know it should work in any system.